src/main/java/study/racingcar/History.java

```diff
@@ -0,0 +1,35 @@
+package study.racingcar;
+
+import java.util.Comparator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+
+public class History {
+
+    private final LinkedList<Map<Car,String>> history;
+
+    public History(LinkedList<Map<Car, String>> result) {
+        this.history = result;
+    }
+
+    public List<Map<Car, String>> history() {
+        return history;
+    }
+
+    public List<String> winners() {
+        Map<Car, String> map = this.history.getLast();
+
+        String maxValue = map.values().stream()
+            .max(Comparator.comparing(String::length))
+            .orElseThrow(() -> new IllegalArgumentException("참여자를 찾을 수 없습니다."));
+
+        return map.entrySet().stream()
+            .filter(carStringEntry -> carStringEntry.getValue().equals(maxValue))
```

Comment:
현재 구조와는 조금 다르니 참고만 해주세요
자동차가 누적한 이동한 거리를 알고 있다면 아래와 같이 변경할 수 있는데요
```suggestion
            .filter(car-> car.samePosition(maxValue))
```
getter로 값을 꺼내어 비교하는 것이 아닌 객체에게 메시지를 보낼 수 있습니다
관련해서 `디미터 법칙`, `Tell Dont Ask` 를 찾아 보시면 좋을 것 같아요

Reply:
이름이 `carStrungEntry` 의미가 전달이 되지 않은 것 같습니다 🥲
![Screenshot 2023-11-13 at 10 55 51 PM](https://github.com/next-step/java-racingcar/assets/52321403/1536e8d9-e360-4a51-8d78-b9ae5c85692a)
이처럼 Map 객체라서 `getValue`밖에 사용할 수가 없었습니다.
하지만 덕분에`디미터 법칙`, `Tell Dont Ask`라는 좋은 개념을 알게 되었습니다. 🙇